src/integrationTest/java/com/mongodb/spark/sql/connector/read/AbstractMongoStreamTest.java

```diff
@@ -327,7 +335,7 @@ void testStreamPublishFullDocumentOnlyHandlesCollectionDrop(
   }
 
   @ParameterizedTest
-  @ValueSource(strings = {"SINGLE", "MULTIPLE", "ALL"})
+  @ValueSource(strings = {"SINGLE", "MULTIPLE"})
```

Comment:
This fixes a bug where it was checking the writes against the source db and not the sink db!  As you cannot test reading from all collections and writing to that db at the same time. I opted to remove the ALL step. Added SPARK-429 to track.